src/components/SearchView.tsx

```diff
@@ -1,47 +1,51 @@
-import SearchResult from '@components/search/SearchResult'
-import { BusinessType, DisplayableBusiness } from '@futureproof/typings'
-import React, { useState } from 'react'
-import { FlatList, SafeAreaView, StyleSheet } from 'react-native'
+import SearchResult from "@components/Search/SearchResult";
+import { BusinessType, DisplayableBusiness } from "@futureproof/typings";
+import React, { useState } from "react";
+import { FlatList, ScrollView, StyleSheet } from "react-native";
 
-const TEST_BUSINESSES_DATA : DisplayableBusiness[] = [
+const TEST_BUSINESSES_DATA: DisplayableBusiness[] = [
   {
-    id: '1',
-    name: 'Starbucks',
-    profileText: 'This is a test business',
+    id: "1",
+    name: "Starbucks",
+    profileText: "This is a test business",
     sustainabilityScore: 100,
     customerScore: 65,
     type: BusinessType.Cafe,
-    profilePicture: 'https://upload.wikimedia.org/wikipedia/en/thumb/d/d3/Starbucks_Corporation_Logo_2011.svg/1200px-Starbucks_Corporation_Logo_2011.svg.png'
-  }
-]
+    profilePicture:
+      "https://upload.wikimedia.org/wikipedia/en/thumb/d/d3/Starbucks_Corporation_Logo_2011.svg/1200px-Starbucks_Corporation_Logo_2011.svg.png",
+  },
+];
 
-const SearchView = () => {
-  const [selectedId, setSelectedId] = useState(null)
+const SearchView = ({ navigation }: { navigation: any }) => {
```

Comment:
Can this component make a GraphQL query to get results

Reply:
Not my section.